vonage/assets/logs/vonage.yaml

```diff
@@ -0,0 +1,147 @@
+id: vonage
+# See app_id in your integration's manifest.json file to learn more:
+# https://docs.datadoghq.com/developers/integrations/check_references/#manifest-file
+metric_id: vonage
+# If for some reason id must be different than app_id, add the app_id in this field instead.
+# If id and app_id already match, this field can be left blank.
+backend_only: false
+facets:
+  - description: Date/Time when the message was delivered/failed to deliver.
+    facetType: list
+    groups:
+      - Vonage
+    name: date_end
+    path: date_end
+    source: log
+    type: string
+  - description: Start date for an initiated call
+    facetType: list
+    groups:
+      - Vonage
+    name: date_start
+    path: date_start
+    source: log
+    type: double
+  - facetType: list
+    groups:
+      - Vonage
+    name: error_code
+    path: error_code
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - Vonage
+    name: id
+    path: log_id
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - Vonage
+    name: Log Level
+    path: log_level
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - Vonage
+    name: message
+    path: message
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - Vonage
+    name: product
+    path: product
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - Vonage
+    name: role
+    path: role
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - Vonage
+    name: status
+    path: status
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - Vonage
+    name: Threads
+    path: message_thread
+    source: log
+    type: double
+  - facetType: list
```

Comment:
I don't think you need these `facetType: list` lines.  For example, this section would be:
```  
  - groups:
    - Web Access
  name: ClientIP
  path: network.client.ip
  source: log 
```
and similar formatting for all the other facets.

Reply:
Resolved, fixed in previous commits.